kobo/hub/xmlrpc/worker.py

```diff
@@ -164,7 +164,7 @@ def fail_task(request, task_id, task_result):
 @validate_worker
 def set_task_weight(request, task_id, weight):
     task = Task.objects.get_and_verify(task_id=task_id, worker=request.worker)
-    task.setWeight(weight)
```

Comment:
What's the deal with this - is it more code which would always have crashed?

Could we please have an issue filed for that too and mentioned in the commit message?

If you have a look at https://github.com/release-engineering/kobo Changelog, you can see there's "bug fixes" section on each release.  Since you're not updating these yourself, you should be putting enough info commit messages so that someone else could do it for you later to generate an accurate changelog.  (I've done all the recent releases and I review what's changed by looking at commit messages.)

Reply:
@rohanpm I'll create an issue, but one good thing of GitHub are milestones, we should create one for the next version and we could add PRs or Issues to it. This will make it easier to find what was done.

Reply:
I had a look at milestones but I'm not sure how to effectively make use of it for releases.  I see that I can create a milestone and put issues and pull requests into it, but I don't think I can use that to reliably find out what was done for a release...

At kobo 0.8.0 release time, we will seek to release latest master and we will have a tag 0.7.0 for previous release, so we can precisely find out everything which was changed between the two.  So we have git, as the primary data source.  If there's also a milestone representing that release, isn't that just a secondary and less accurate source for the same info?  It seems to rely on somebody clicking buttons to attach PRs/issues to the milestones, and what's done there might not match what actually goes into the repo.  So it does not free the release master from checking what changed in git.